anagrafica/permessi/funzioni.py

```diff
@@ -83,6 +91,7 @@ def permessi_ufficio_soci_unita(sede):
     """
     sede_qs = sede.queryset_modello()
     return [
+        (RUBRICA_UFFICIO_SOCI_UNITA, sede.espandi(includi_me=True, pubblici=True)),
         (GESTIONE_SOCI,     sede_qs),
         (ELENCHI_SOCI,      sede_qs),
     ]
```

Comment:
@luca-dex in https://github.com/CroceRossaItaliana/jorvik/pull/405#issuecomment-262228203

> i presidenti dovrebbero poter vedere tutte le rubriche che gli competono (ad esempio quelle di tutti i delegati di area, degli uffici soci, ecc) ma non vedono nulla

Questo e' inoltre vero generalmente per ogni potere derivante dalle deleghe -- i Presidenti hanno infatti tutti i poteri, e delegano' anche a degli individui che li aiutano nella gestione del Comitato, da qui i 'delegati'. Inoltre l'Ufficio Soci e' quello che ha accesso e gestisce tutti i soci del Comitato, quindi ha senso che possa accedere alle rubriche per comodita', a mio avviso.